alembic/versions/100ed5f27d44_add_soft_delete_to_events_model.py

```diff
@@ -0,0 +1,208 @@
+"""add soft delete to events model
+
+Revision ID: 100ed5f27d44
+Revises: 8647af58767d
+Create Date: 2019-02-05 17:35:40.230515
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '100ed5f27d44'
+down_revision = '8647af58767d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('locations',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('abbreviation', sa.String(), nullable=False),
+    sa.Column('country', sa.Enum('Uganda', 'Kenya', 'Nigeria', name='countrytype'), nullable=True),
+    sa.Column('time_zone', sa.Enum('EAST_AFRICA_TIME', 'WEST_AFRICA_TIME', name='timezonetype'), nullable=True),
+    sa.Column('image_url', sa.String(), nullable=True),
+    sa.Column('state', sa.Enum('active', 'archived', 'deleted', name='statetype'), nullable=True),
+    sa.PrimaryKeyConstraint('id')
```

Comment:
Please generate an Alembic file which captures only your changes and it should revise from the latest revision to avoid generating a multiple heads error.  Since you added one field to the events model, your alembic file should only capture that.

Reply:
@michael-basweti  I have implemented the feedback.